app.js

```diff
@@ -0,0 +1,24 @@
+$( document ).ready(function(event) {
+
+	$('.readmore').click(function(event){
+		event.preventDefault();
+		$('.readmore').hide();
+		$('#show-this-on-click').slideDown();
+		$('.readless').removeClass('hide');
+	});
+
+	$('.readless').click(function(event){
+		event.preventDefault();
+		$('#show-this-on-click').slideUp();
+		$('.readless').addClass('hide');
```

Comment:
Dont need ```addClass('hide')```, does exactly the same thing as ```hide()``` except you have to add your own css class.